sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -624,6 +630,52 @@ class EthService(
     }
   }
 
+  @RpcMethod("debug_traceCall")
+  @RpcOptionalParameters(1)
+  def traceCall(tx: TransactionArgs, tag: String, config: TraceOptions): Any = {
```

Comment:
This function also accepts "tag", we need to make sure this also works with the new "pending" block.

Nit pick: I would prefer a different variable name than `tx`, because it is _not_ a transaction.

Reply:
Davide, pls keep in mind this PR: https://github.com/HorizenOfficial/Sidechains-SDK/pull/714
has been merged to dev_evm this morning

Reply:
Rebase with the changes introduced with the PR https://github.com/HorizenOfficial/Sidechains-SDK/pull/714
Python test updated with _traceCall_ method call using the _pending_ tag